aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -658,10 +658,30 @@ std::tuple<Tensor, Tensor, Tensor, Tensor> native_decoder_only_multi_head_attent
 }
 
 int64_t _fused_sdp_choice_cpp(const Tensor& query_, const Tensor& key, const Tensor& value,
-        const c10::optional<Tensor>& attn_mask_, double dropout_p, bool need_attn_weights, bool is_causal){
+        const c10::optional<Tensor>& attn_mask_, double dropout_p, bool is_causal){
   return static_cast<int64_t>(sdp::SDPBackend::math);
 }
 
+//  !!!!!! TODO: THIS NEEDS TO BE REMOVED BUT PEOPLE HAVE TRAINED THEIR MODELS
+//  WITH THIS OP BUILTIN !!!!!!
+std::tuple<Tensor, Tensor> _scaled_dot_product_attention(
+    const Tensor& query_,
+    const Tensor& key,
+    const Tensor& value,
+    const c10::optional<Tensor>& attn_mask_,
+    double dropout_p,
+    bool need_attn_weights,
+    bool is_causal) {
+  if (!need_attn_weights) {
+    return std::make_tuple(
```

Comment:
@cpuhrsch I think this implementation of the underscored version should match until we get get rid of the aten op